entrypoint

```diff
@@ -87,6 +87,7 @@ script/dev:db:downgrade) shift; exec yarn run dev:db:downgrade "${@}";;
 script/dev:apply) shift; exec yarn run dev:apply "${@}";;
 script/dev:delete) shift; exec yarn run dev:delete "${@}";;
 script/dev:verify) shift; exec yarn run dev:verify "${@}";;
+script/dev:templates) shift; exec yarn run dev:templates "${@}";;
```

Comment:
this is to dump k8s templates with substituted variables into a file for debug purposes